extras/docker/Dockerfile.buildtest

```diff
@@ -12,7 +12,7 @@ COPY . $BUILDDIR
 # prepare the system
 RUN true \
  && dnf -y install \
-           git autoconf automake gcc libtool make file \
+           git autoconf automake gcc libtool make \
```

Comment:
This was actually not a typo. `./configure` prints a warning if `/usr/bin/file` is not found. That said it's not a blocker, since it's only a warning....